views/admin.php.inc

```diff
@@ -1,183 +1,173 @@
-                                if (!is_null($display_opts)) {
-                                    if ($display_opts->show_onfrontpage) {
-                                        $checked = true;
-                                    }
-                                }
-                                ?>
-                                <input type="checkbox" class="checkbox show_specific" name="show_onfrontpage"
-                                    id="show_onfrontpage" value="1"
-                                    <?php echo ($checked)?'checked':'';?> />
-                            </div>
-                        </div>
-
-                        <div class="form-group col-lg-12">
-                            <label for="show_oncategory" class="col-lg-6 control-label">Show on category pages</label>
-                            <div class="col-lg-6 control-label ">
-                                <?php
-                                $checked = false;
-                                if (!is_null($display_opts)) {
-                                    if ($display_opts->show_oncategory) {
-                                        $checked = true;
-                                    }
-                                }
-                                ?>
-                                <input type="checkbox" class="checkbox show_specific" name="show_oncategory" id="show_oncategory" value="1"
-                                    <?php echo ($checked)?'checked':'';?>  />
-                            </div>
-                        </div>
-                        <div class="form-group col-lg-12">
-                            <label for="show_oncustom" class="col-lg-6 control-label">Show on pages:</label>
-                            <div class="col-lg-6 control-label">
-                                <?php if (isset($display_opts->show_oncustom) && !empty($display_opts->show_oncustom)) : ?>
-                                    <?php $whitelist = json_decode($display_opts->show_oncustom) ?>
-                                    <textarea class="form-control show_specific" name="show_oncustom"
-                                        id="show_oncustom" cols="30" rows="10"><?php foreach ($whitelist as $page) { echo $page."\r\n"; } ?></textarea>
-                                <?php else : ?>
-                                    <textarea class="form-control show_specific" name="show_oncustom" id="show_oncustom" cols="30" rows="10"></textarea>
-                                <?php endif; ?>
-                                <br>
-                                <p style="text-align: justify;">
-                                Add URLs to pages in which you would like to show the widget.<br>
-                                Put each URL in a new line.
-                                </p>
+{   ob_start();
+    ?><script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
```

Comment:
No need to add jquery cdn since backdrop cms loads up jquery by default

Reply:
Removed.  I had removed it, but ending up adding back in when trying to troubleshoot an issue and forgot to remove it.